### DISCUSSION TITLE
我也想在電腦養寵物之我只有鵝

### DISCUSSION THREAD

**Discussion by Brownnnn78** [1101-GS4719-C/Python-Community-GS4719-C#1182] [General]
延續上一位同學推薦的VScode養寵物，我也要推薦各位下載這個讓你電腦變慢的鵝，他會直接在你桌面亂跑、叼走你的滑鼠游標、給你看真的鵝或是留言給你!
這程式叫做"Desktop Goose" 
下載連結:
https://samperson.itch.io/desktop-goose?fbclid=IwAR1Zv-nAefBz64tSFUDmz-he2Wgq1M3H-k1XArkCM54A_OL4t7Fm_Wiqvsc
下載的時候他會說要收錢，但只要按一個"讓我下載!!"的按鈕(他是說寫英文啦，純粹個人翻譯)，就可以下載了!
下載完後按資料夾中的Desktop Goose就可以叫鵝出來，按很多次就會鵝鵝鵝鵝，記得開聲音!
祝你們都鵝鵝鵝鵝鵝

**Comment by Lipunpun** [comment]
我之前家裡也有養鵝，在電腦裡面養有很多好處，其中鵝的排泄物我真的受不了，幾乎每小時都會拉，他又不像是貓狗會定點，所以我認為這個程式對於我這種人十分受用！
